types/screener.d.ts

```diff
@@ -0,0 +1,66 @@
+//
+// Typings for `screener-runner`
+//
+
+type ScreenerRunnerKeys = {
+  alt: string
+  control: string
+  enter: string
+  escape: string
+  return: string
+  shift: string
+  tab: string
+  leftArrow: string
+  upArrow: string
+  rightArrow: string
+  downArrow: string
+}
+
+type ScreenerRunnerSteps = {
+  /** This will click on the first element matching the provided css selector. */
+  click(selector: string): ScreenerRunnerSteps
+
+  /** This will capture a visual snapshot. */
+  snapshot(name: string): ScreenerRunnerSteps
```

Comment:
has an optional parameter as well:
```suggestion
  snapshot(name: string, options?: any): ScreenerRunnerSteps
```